### PR TITLE
feat:Remove group usage(RWA-722)

### PR DESCRIPTION
**Pull request by dharmendrak**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RWA-722

### Change description ###

Remove group field

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/SendMessageTest.java [hunk 1]
```diff
@@ -123,16 +122,6 @@ public void transition_creates_a_task_with_default_due_date() {
             });
 
         String taskId = taskIdResponse.get();
-
-        Response result = camundaApiActions.get("/task/{task-id}/identity-links?type=candidate",
-            taskId,
-            new Headers(authenticationHeaders)
-        );
-
-        result.then().assertThat()
```
Comment (BenNzewi) [inline src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/SendMessageTest.java:132]: we can leave the assertion on status. That qualifies the test.

File: src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/SendMessageTest.java [hunk 2]
```diff
@@ -195,25 +183,6 @@ public void transition_creates_a_task_with_due_date() {
...
-            .until(() -> {
-
-
-                Response result = camundaApiActions.get("/task/{task-id}/identity-links?type=candidate",
-                    taskId,
-                    new Headers(authenticationHeaders)
-                );
-
-                result.then().assertThat()
-                    .statusCode(HttpStatus.OK.value())
```
Comment (BenNzewi) [inline src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/SendMessageTest.java:211]: we can leave the assertion on status. That qualifies the test.

**Review by mkayad — COMMENTED** [review]
LGTM

**Review by mkayad — APPROVED** [review]
LGTM

**Pull request merged**